pyfar/classes/audio.py

```diff
@@ -736,12 +806,23 @@ def times(self):
     @property
     def frequencies(self):
         """Frequencies of the discrete signal spectrum."""
-        return np.atleast_1d(fft.rfftfreq(self.n_samples, self.sampling_rate))
+        if self.complex:
+            # assume the time domain data were complex-valued
+            # such that we need a two-sided Fourier spectrum
+
+            return np.atleast_1d(fft.fftfreq(self.n_samples,
+                                             self.sampling_rate))
+        else:
+            return np.atleast_1d(fft.rfftfreq(self.n_samples,
+                                              self.sampling_rate))
 
     @property
     def n_bins(self):
         """Number of frequency bins."""
-        return fft._n_bins(self.n_samples)
+        if self.complex:
+            return self.n_samples
+        else:
```

Comment:
I would suggest to also pass self.complex to fft._n_bins(). This would make it a bit more consistent for my taste.

Reply:
Done.